python/orion_finance_sdk_py/erc20.py

```
@@ -0,0 +1,170 @@
+    ).build_transaction(
+        {
+            "from": account.address,
+            "nonce": nonce,
+            "gasPrice": w3.eth.gas_price,
+        }
+    )
+    signed = account.sign_transaction(tx)
+    tx_hash = w3.eth.send_raw_transaction(signed.raw_transaction)
+    receipt = w3.eth.wait_for_transaction_receipt(tx_hash)
+    if receipt["status"] != 1:
+        raise Exception(f"ERC-20 transfer failed with status: {receipt['status']}")
```

Ruff (0.16.1)

[warning] Create your own exception

(TRY002)